#!/usr/bin/env python

PACKAGE = "cvc_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("delta_a",             double_t, 0, "Default: 0.4",    0.4,  0.1,  3.0)
gen.add("delta_p",             double_t, 0, "Default: 1.2",    1.2,  1.0,  12)
gen.add("delta_r",             double_t, 0, "Default: 0.2",    0.2,  0.1,  2)

exit(gen.generate(PACKAGE, "cvc_ros", "cvc_ros_"))